README.md

```diff
@@ -29,8 +29,8 @@ A netbox plugin providing floorplan mapping capability for locations and sites
 
 The plugin is available as a Python package in pypi and can be installed with pip  
 
+
 ```
-source /opt/netbox/venv/bin/activate
```

Comment:
I tend to favor leaving all necessary instructions in place to up the odds of success for someone installing this plugin as their first plugin. If the deletion is about the pathname being possibly invalid, I think we can go with `${NETBOX_HOME}/venv/bin/activate` or similar.